typescript/src/schema/proto.ts

```diff
@@ -167,7 +167,9 @@ function pagingField(
   const serviceName =
```

Comment:
Not directly related to this PR, but while you're here, let's maybe rename these variables to `serviceNameException` and `methodNameException`? It's weird for a variable named like `methodName` to be boolean.

Reply:
yes, of course. That makes sense!

Reply:
done.